### DISCUSSION TITLE
Which CSI is better for prometheus to create persistent volume in Azure

### DISCUSSION THREAD

**Discussion by uvaisibrahim** [prometheus/prometheus#10416] [Q&A]
Hi Team,

Which CSI drivers is more better to create persistent volume for prometheus in a  kubernetes cluster in Azure(not AKS) in terms of performance and cost.

- Blob
- Azure Disk
- Azure File

Thanks,

**Comment by zak-afk** [comment]
hello, did you find the answer ?
